CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 We follow [Semantic Versioning](http://semver.org/) as a way of measuring stability of an update. This
 means we will never make a backwards-incompatible change within a major version of the project.
 
+## v0.2.0
+
+- added Cherrypy server hooks into `build_bot()` and `run_until_dead()` to create a json endpoint for use as a heartbeat check (by @itsthejoker)
```

Comment:
What about shortening this line to something like:

> Added heartbeat web service to check status of a bot (by @itsthejoker)

The changelog is intended to be very high level summary and what is currently written seems like there's too much technical detail, a problem I have all the time. Or is there are reason for adding that detail?